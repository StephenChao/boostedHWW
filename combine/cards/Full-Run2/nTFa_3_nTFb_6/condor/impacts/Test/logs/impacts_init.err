Run: [FATAL] Socket timeout:  (source)
tar: CMSSW_11_3_4.tgz: Cannot open: No such file or directory
tar: Error is not recoverable: exiting now
rm: cannot remove 'CMSSW_11_3_4.tgz': No such file or directory
/srv/condor_exec.exe: line 22: cd: CMSSW_11_3_4/src/: No such file or directory
