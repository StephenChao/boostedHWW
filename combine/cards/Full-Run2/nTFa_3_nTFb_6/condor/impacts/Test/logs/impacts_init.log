000 (17168632.000.000) 2024-05-12 14:24:57 Job submitted from host: <192.170.231.212:9618?addrs=192.170.231.212-9618&alias=login-el7.uscms.org&noUDP&sock=schedd_8741_2d6c>
...
040 (17168632.000.000) 2024-05-12 14:25:32 Started transferring input files
	Transferring to host: <192.168.101.111:43277?CCBID=128.142.194.25:9618%3faddrs%3d[2001-1458-301-cd--100-364]-9618+128.142.194.25-9618%26alias%3dvocms0813.cern.ch%26noUDP%26sock%3dcollector9635#11028288%20131.225.189.210:9618%3faddrs%3d[2620-6a-0-8421--cd]-9618+131.225.189.210-9618%26alias%3dcmssrvz03.fnal.gov%26noUDP%26sock%3dcollector9693#370766&PrivNet=tds401.pic.es&addrs=[2001-67c-1148-301--401]-43277+192.168.101.111-43277&alias=tds401.pic.es&noUDP>
...
001 (17168632.000.000) 2024-05-12 14:25:48 Job executing on host: <192.168.101.111:42711?CCBID=128.142.194.25:9618%3faddrs%3d[2001-1458-301-cd--100-364]-9618+128.142.194.25-9618%26alias%3dvocms0813.cern.ch%26noUDP%26sock%3dcollector9623#11052824%20131.225.189.210:9618%3faddrs%3d[2620-6a-0-8421--cd]-9618+131.225.189.210-9618%26alias%3dcmssrvz03.fnal.gov%26noUDP%26sock%3dcollector9667#355099&PrivNet=tds401.pic.es&addrs=[2001-67c-1148-301--401]-42711+192.168.101.111-42711&alias=tds401.pic.es&noUDP>
...
040 (17168632.000.000) 2024-05-12 14:25:48 Finished transferring input files
...
006 (17168632.000.000) 2024-05-12 14:25:56 Image size of job updated: 2604868
	34  -  MemoryUsage of job (MB)
	59872  -  ResidentSetSize of job (KB)
	34566  -  ProportionalSetSize of job (KB)
...
006 (17168632.000.000) 2024-05-12 14:47:11 Image size of job updated: 2604868
	59  -  MemoryUsage of job (MB)
	59872  -  ResidentSetSize of job (KB)
	34566  -  ProportionalSetSize of job (KB)
...
040 (17168632.000.000) 2024-05-12 14:47:11 Started transferring output files
...
040 (17168632.000.000) 2024-05-12 14:47:11 Finished transferring output files
...
007 (17168632.000.000) 2024-05-12 14:47:12 Shadow exception!
	Error from slot1_2@glidein_9_66622237@tds401.pic.es: STARTER at 192.168.101.111 failed to send file(s) to <192.170.231.212:9618>: |Error: reading from file /tmp/glide_L5EQYd/execute/dir_790068/higgsCombine_initialFit_impacts.MultiDimFit.mH125.root: (errno 2) No such file or directory; SHADOW failed to receive file(s) from <193.109.175.203:42261>
	421  -  Run Bytes Sent By Job
	3829806  -  Run Bytes Received By Job
...
012 (17168632.000.000) 2024-05-12 14:47:12 Job was held.
	Error from slot1_2@glidein_9_66622237@tds401.pic.es: STARTER at 192.168.101.111 failed to send file(s) to <192.170.231.212:9618>: |Error: reading from file /tmp/glide_L5EQYd/execute/dir_790068/higgsCombine_initialFit_impacts.MultiDimFit.mH125.root: (errno 2) No such file or directory; SHADOW failed to receive file(s) from <193.109.175.203:42261>
	Code 13 Subcode 2
...
013 (17168632.000.000) 2024-05-12 14:53:01 Job was released.
	via condor_release (by user yuzhe)
...
040 (17168632.000.000) 2024-05-12 23:31:00 Started transferring input files
	Transferring to host: <134.158.51.102:36769?CCBID=128.142.194.25:9618%3faddrs%3d[2001-1458-301-cd--100-364]-9618+128.142.194.25-9618%26alias%3dvocms0813.cern.ch%26noUDP%26sock%3dcollector9696#11033631%20131.225.189.210:9618%3faddrs%3d[2620-6a-0-8421--cd]-9618+131.225.189.210-9618%26alias%3dcmssrvz03.fnal.gov%26noUDP%26sock%3dcollector9680#370048&PrivNet=ccwcondor0556.in2p3.fr&addrs=[2001-660-5009-312-134-158-51-102]-36769+134.158.51.102-36769&alias=ccwcondor0556.in2p3.fr&noUDP>
...
001 (17168632.000.000) 2024-05-12 23:31:01 Job executing on host: <134.158.51.102:35482?CCBID=128.142.194.25:9618%3faddrs%3d[2001-1458-301-cd--100-364]-9618+128.142.194.25-9618%26alias%3dvocms0813.cern.ch%26noUDP%26sock%3dcollector9710#11020692%20131.225.189.210:9618%3faddrs%3d[2620-6a-0-8421--cd]-9618+131.225.189.210-9618%26alias%3dcmssrvz03.fnal.gov%26noUDP%26sock%3dcollector9688#371046&PrivNet=ccwcondor0556.in2p3.fr&addrs=[2001-660-5009-312-134-158-51-102]-35482+134.158.51.102-35482&alias=ccwcondor0556.in2p3.fr&noUDP>
...
040 (17168632.000.000) 2024-05-12 23:31:02 Finished transferring input files
...
006 (17168632.000.000) 2024-05-12 23:36:10 Image size of job updated: 4022952
	436  -  MemoryUsage of job (MB)
	703456  -  ResidentSetSize of job (KB)
	445501  -  ProportionalSetSize of job (KB)
...
