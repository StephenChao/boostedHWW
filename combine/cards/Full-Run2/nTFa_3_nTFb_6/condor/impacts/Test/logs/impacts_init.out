WARN $TMPDIR(/tmp/glide_L5EQYd/execute/dir_790068) is not a writable directory setting $TMPDIR = $PWD
running impacts
extracting tar
